bin/combine_process_settings

```diff
@@ -75,18 +75,40 @@ def add_warning_comment(yaml, root_folder, program_name, settings_folder)
   yaml.sub("\n", "\n" + warning_comment)
 end
 
+MINIMUM_LIBYAML_VERSION = '0.2.5' # So that null (nil) values don't have trailing spaces.
+
+def check_libyaml_version
+  if Gem::Version.new(Psych::LIBYAML_VERSION) < Gem::Version.new(MINIMUM_LIBYAML_VERSION)
+    warn <<~EOS
+
+      #{PROGRAM_NAME} error: libyaml version #{Psych::LIBYAML_VERSION} must be at least #{MINIMUM_LIBYAML_VERSION}. Try:
+
+          brew update && upgrade libyaml
```

Comment:
This is a public gem, do we want to make this installation command a bit more generic?

Reply:
I don't quite follow what you're suggesting? Can you follow up in Slack?
Meanwhile I see a typo there--I forgot `brew` in front of `upgrade`. Doh. I'll fix that.